### DISCUSSION TITLE
Rookie question

### DISCUSSION THREAD

**Discussion by emincer** [filamentphp/filament#5593] [General]
Sorry, I'm relatively new to Laravel and totally new to Filament.

I have Companies and Users, users can belong to many companies and I have the usual company_users table that stores the user_id and company_id. This is all working just fine. 

But within each company that user can have one of several different roles such as user, manager, admin. I made a company_account_types table just to store those strings so we can add roles at any time. And I added account_type to the this table.

But I'm getting lost in my relationship structure.  With the following code
`            ->columns([
                TextColumn::make('user_id'),
                TextColumn::make('company_id'),
                TextColumn::make('account_type'),
            ])`

in my CompanyUserResource.php file, I do get a table showing those ids numerically. But how do I use those ids to get the corresponding name? I feel like I'm missing something simple. I tried user_id.name etc but that doesn't do it. And I don't think I need to go into relationship managers for this do I? What am I missing? Is this not the correct structure?

**Comment by pxlrbt** [comment]
I think this was already answered on Discord, right?

**Reply by emincer** [reply]
Yes, thank you
